client/src/components/navbar/navbar.js

```diff
@@ -17,52 +17,32 @@ const renderTooltip = props => {
   return (
     <Tooltip bsPrefix = 'tooltip' {...props} >
       { pathname !== '/'
-          ? 'Click to go Home'
-          : 'Home'
+        ? 'Click to go Home'
+        : 'Home'
       }
     </Tooltip>
   );
 };
 
 function MyNavbar(props) {
   const { currentUser, logOut } = props;
-  return (
-    <Navbar expand = 'lg' sticky = 'top' className = 'mynavbar navbar-dark'>
-      <Navbar.Brand>
-        <Link to = { '/' } className = 'navbar-brand'>
-          <OverlayTrigger
-            placement = 'bottom-start'
-            overlay = { renderTooltip }
-          >
-            <img
-              src = { pbchessLogo }
-              width = '60em'
-              height = '60em'
-              className = 'd-inline-block align-top'
-              alt = 'pbchess logo'
-            />
-          </OverlayTrigger>
-        </Link>
-      </Navbar.Brand>
-      <Navbar.Toggle aria-controls = 'basic-navbar-nav' />
-      <Navbar.Collapse id = 'basic-navbar-nav'>
-        <Nav className = 'ml-auto'>
-          <Nav.Link>
-            <Link to = { '/play' }>
-              Play
-            </Link>
-          </Nav.Link>
-          <Nav.Link>
-            <Link to = { '/learn' }>
-              Learn
-            </Link>
-          </Nav.Link>
-          { currentUser
-            ? (
-              <NavDropdown
-                title = { currentUser.username }
-                id = 'collapsible-nav-dropdown'
-              >
+  return(
+      <Navbar expand = 'lg' sticky = 'top' className = 'mynavbar navbar-dark'>
+        <Navbar.Brand>
+          <Link to = { '/' } className = 'navbar-brand'>
+            <OverlayTrigger placement = 'right-end' overlay = { renderTooltip }>
+              <img src = { pbchessLogo } width = '60em' height = '60em' className = 'd-inline-block align-top' alt = 'pbchess logo'/>
+            </OverlayTrigger>
+          </Link>
+
+        </Navbar.Brand>
+        <Navbar.Toggle aria-controls="basic-navbar-nav"/>
+        <Navbar.Collapse id="basic-navbar-nav">
+          <Nav className="ml-auto">
+            <Nav.Item><Nav.Link href="/play">Play</Nav.Link></Nav.Item>
```

Comment:
@kumar-tushar By not using `Link` from `react-router-dom`, it defeats the purpose of SPA and faster loads. As you can see Nav.Link's href introduces a delay in page load!

Reply:
@fave77 Hello Sir, I have made some changes, Please have a look.

Reply:
https://drive.google.com/file/d/18EV7j18GiFHd6R9bpXFjwGwxZBsW7tj1/view?usp=sharing